src/client/client.c

```diff
@@ -1531,6 +1531,34 @@ static const char* client_usage_msg[][70] = {
 
 };
 
+static void showPeerPEM(WOLFSSL* ssl)
```

Comment:
Is client.c here the same client.c from the wolfssl repo? If so, consider adding wolfssl as a git submodule or something so that we don't have to keep these in sync.

Reply:
Yes, it currently is using the client from wolfssl. The script certs/renew.sh pulls it in, updating the client code when the certificates are. Eventually it may diverge from the example client in wolfSSL.